tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -175,6 +176,21 @@ sub run {
             $playbook_configs{spn_application_password} = get_var('AZURE_SPN_APP_PASSWORD', get_required_var('_SECRET_AZURE_SPN_APP_PASSWORD'));
         }
     }
+
+    my @addons = grep { defined $_ && $_ } split(/,/, get_var('SCC_ADDONS'));
+    # This implementation has a known limitation
+    # if SCC_ADDONS has two or more elements (like "ltss,ltss_es")
+    # only the last one will be added to the playbook argument.
+    foreach my $addon (@addons) {
+        my $name;
+        # Keep the code simple by only support ltss addons,
+        # it simplify version calculation.
+        $name = get_addon_fullname($addon) if ($addon =~ 'ltss');
+        if (length $name) {
```

Comment:
`if ($name)` should be good enough.

Reply:
yes, I get it from https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/602a746062ede11b555eb2f62da4bbed7b3fdead/lib/registration.pm#L322